frontend/src/pages/MessagingQueues/MQDetailPage/MQDetailPage.tsx

```diff
@@ -78,10 +81,21 @@ function MQDetailPage(): JSX.Element {
 			</div>
 			<div className="messaging-queue-main-graph">
 				<MessagingQueuesConfigOptions />
-				<MessagingQueuesGraph />
+				{selectedView === MessagingQueuesViewType.consumerLag.value ? (
```

Comment:
NIT: if we can make use of switch statement here and render things ? would help to scale if different components come in for different views later.